contrib/automation_tests/orbit_tracks.py

```diff
@@ -18,7 +18,7 @@ def main(argv):
         Capture(),
         # "sdma0" is not present on the DevKits, instead there is "vce0", so this tests for "sdma0 or vce0"
         MatchTracks(expected_names=[
-            "Scheduler", ("sdma0", "vce0"), "gfx", "All Threads", "hello_ggp_stand"],
+            "Scheduler", ("*sdma0*", "*vce0*"), "gfx", "All Threads", "hello_ggp_stand"],
```

Comment:
Is this matching done on purpose? Iirc the track names are "sdma0" or "vce0" exactly. Doesn't really matter though I think.

Reply:
I wasn't 100% sure about the DevKits, so I thought it's better to be safe here. This is basically equivalent to the old behavior.